README.md

```
@@ -1,58 +1,326 @@
+- **Admin Panel**: [jordanpartridge.us/admin](https://jordanpartridge.us/admin)
+- **GitHub**: [github.com/jordanpartridge/jordanpartridge.us](https://github.com/jordanpartridge/jordanpartridge.us)
+- **Forge Deployment**: [Dashboard](https://forge.laravel.com/servers/820904/sites/2398933)
+
+## 📝 License
+
+This project is open-sourced software licensed under the [MIT license](https://opensource.org/licenses/MIT).
+
+---
+
+**Built with ❤️ by Jordan Partridge**  
+*Laravel Developer | Cycling Enthusiast | Fat Bike Corps*
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)